src/APIInterfaces/BackendAPI/Requests/CourseRequests.ts

```diff
@@ -671,6 +731,30 @@ export const postConfirmAttachmentUpload = async (options: PostConfirmAttachment
     }
 };
 
+export const postGenericConfirmAttachmentUpload = async (options: PostGenericConfirmAttachmentUploadOptions): Promise<AxiosResponse<BackendAPIResponse>> => {
```

Comment:
Doesn't matter now but I feel like logic should be split out, this way only the api call itself is in here